overlord/auth/auth.go

```diff
@@ -248,3 +269,49 @@ func (ac *authContext) UpdateUser(user *UserState) error {
 
 	return UpdateUser(ac.state, user)
 }
+
+// StoreID returns the store id according to system state or
+// the fallback one if the state has none set (yet).
+func (ac *authContext) StoreID(fallback string) (string, error) {
+	if storeID := os.Getenv("UBUNTU_STORE_ID"); storeID != "" {
+		return storeID, nil
+	}
+	var storeID string
```

Comment:
strange that you'd do it this way and not move the getenv out of the if. worried about getenv leaving it dirty?